docs/noencap-hybrid-modes.md

```diff
@@ -0,0 +1,174 @@
+# NoEncap and Hybrid Traffic Modes of Antrea
+
+Besides the default `Encap` mode, in which Pod traffic across Nodes will be
+encapsulated and sent over tunnels, Antrea also supports `NoEncap` and `Hybrid`
+traffic modes. In `NoEncap` mode, Antrea does not encapsulate Pod traffic, but
+relies on the Node network to route the traffic across Nodes. In `Hybrid` mode,
+Antrea encapsulates Pod traffic when the source Node and the destination Node
+are in different subnets, but does not encapsulate when the source and the
+destination Nodes are in the same subnet. This document describes how to
+configure Antrea with the `NoEncap` and `Hybrid` modes.
+
+## Hybrid Mode
+
+Let us start from `Hybrid` mode which is simpler to configure. `Hybrid` mode
+does not encapsulate Pod traffic when the source and the destination Nodes are
+in the same subnet. Thus it requires the Node network allow Pod IP addresses
+being sent out from the Nodes' NICs. This requirement is not supported in all
```

Comment:
it requires the Node network to allow Pod IP addresses to be sent out

or

it requires that the Node network allow Pod IP addresses to be sent out

Reply:
Updated.